internal/cli/atlas/deployments/search/indexes/create.go

```diff
@@ -38,10 +39,12 @@ import (
 const (
 	namePattern        = "^[a-zA-Z0-9][a-zA-Z0-9-]*$"
 	connectWaitSeconds = 10
-	createTemplate     = "Search index created with ID {{.IndexID}}\n"
+	createTemplate     = "Search index created with ID: {{.IndexID}}\n"
 	notFoundState      = "NOT_FOUND"
 )
 
+var ErrSearchIndexDuplicated = errors.New("search index is duplicated")
```

Comment:
[optional] wdyt about:
```suggestion
var ErrSearchIndexAlreadyExists = errors.New("search index already exists")
```

Reply:
based myself on index API output, no strong opinion though, wdyt? 
```shell

Error: [https://cloud.mongodb.com/api/atlas/v2/groups/623202922f697375cac9414a/clusters/Cluster07293/fts/indexes](https://www.google.com/url?q=https://cloud.mongodb.com/api/atlas/v2/groups/623202922f697375cac9414a/clusters/Cluster07293/fts/indexes&sa=D&source=docs&ust=1695219566833407&usg=AOvVaw36Yeiu-SptRWBb-WkOaAH4) POST: HTTP 400 (Error code: "ATLAS_FTS_DUPLICATE_INDEX") Detail: Duplicate Index. Reason: Bad Request. Params: []
``
`